.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "parser": "typescript",
+  "printWidth": 100,
```

Comment:
Is this width up for disquisition? I find it sometimes does more harm then good when splitting things after 100 chars.

Reply:
I just copied this from relationsbanken, and really don't know if a (consious) decision has been made for 100 characters (I assume someone has).

Prettier actually has a strong opinion of not using more than 80 characters (have a look at this link: https://github.com/prettier/prettier/issues/2375)

Maybe ask @jakobe ?

Reply:
So the question is more how hard we want to follow Prettier. Should we do the recommended 80 characters then?

But lets hear the opinion of @jakobe and Alexander.